src/overlays/actors/ovl_En_Blkobj/z_en_blkobj.c

```diff
@@ -27,23 +31,157 @@ const ActorInit En_Blkobj_InitVars = {
     (ActorFunc)EnBlkobj_Update,
     (ActorFunc)EnBlkobj_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Blkobj/func_809C2060.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Blkobj/EnBlkobj_Init.s")
+static InitChainEntry sInitChain[] = {
+    ICHAIN_VEC3F(scale, 1, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneForward, 800, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneScale, 200, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneDownward, 300, ICHAIN_STOP),
+};
+
+UNK_TYPE D_809C2590[] = {
```

Comment:
are these correct?

Reply:
These look like display lists, aka `Gfx[]` with the proper macros for each command (2 words per command).

Reply:
Gfx macros also need to be used here instead of raw words.